packages/slate-tables/src/core/onKeyDown.ts

```diff
@@ -47,27 +47,32 @@ function onUpPress(editor: TablesEditor): Point | undefined {
 
     const { activeCell, matrix } = traverse;
 
-    if (activeCell.row.isFirst) {
-        return Editor.before(editor, matrix.path, { unit: 'block' });
-    }
+    const [cellStart] = Editor.edges(editor, activeCell.path);
```

Comment:
It doesn't make sense to use `Editor.edges()` and only keep a part of the result. 

Just use `Editor.start()` directly.

From `slate` sources:

```ts
  /**
   * Get the start and end points of a location.
   */
  edges(editor, at) {
    return [Editor.start(editor, at), Editor.end(editor, at)];
  },
```